src/Streamly/Internal/FS/Event/Linux.hs

```diff
@@ -699,10 +671,11 @@ addToWatch cfg@Config{..} watch0@(Watch handle wdMap) root0 path0 = do
     -- handle it. For example, if it is a dir create the application can read
     -- the dir to scan the files in it.
     --
-    let root = removeTrailingSlash root0
-        path = removeTrailingSlash path0
-        absPath = appendPaths root path
-    putStrLn $ "root = " ++ utf8ToString root ++ " path = " ++ utf8ToString path ++ " absPath = " ++ utf8ToString absPath
+    let absPath = combinePathRep prep
+        showPathRep (root, Nothing) = "root = " ++ Path.toString root
+        showPathRep (root, Just sub) =
+            "root = " ++ Path.toString root ++ " sub = " ++ Path.toString sub
+    putStrLn $ showPathRep prep ++ " absPath = " ++ Path.toString absPath
```

Comment:
Not sure why we have this print statement here. This may have been left from debug code. Should be commented.